source/common/config/README.md

```diff
@@ -9,39 +9,51 @@ you can mostly forget the filesystem/REST/gRPC distinction, and you can
 especially forget about the gRPC flavors. All of that is specified in the
 bootstrap config, which is read and put into action by ClusterManagerImpl.
 
-Note that there can be multiple active gRPC subscriptions for a single resource
```

Comment:
@wgallagher in a follow up change can we move this doc to source/docs where our dev docs go and/or figure out if this should be merged into the RST docs?